cmd/perf-tests.go

```diff
@@ -422,3 +408,37 @@ func siteNetperf(ctx context.Context, duration time.Duration) madmin.SiteNetPerf
 		TotalConn:       uint64(connectionsPerPeer),
 	}
 }
+
+// perfNetRequest - reader for http.request.body, result should be a pointer if it need
+func perfNetRequest(ctx context.Context, timeout time.Duration, deploymentID string, reqPath string, reader io.Reader, result interface{}) (enpoint string, err error) {
```

Comment:
```suggestion
func perfNetRequest(ctx context.Context, timeout time.Duration, deploymentID, reqPath string, reader io.Reader, result any) (endpoint string, err error) {
```

Why are you sending a timeout? Wouldn't attaching the timeout to the context do the same?

Reply:
Timeout should be used to http.Request. Shouldn't include get the client.